src/Integration/Binding/RuleSetReferenceChecker.cs

```diff
@@ -74,15 +74,15 @@ private RuleSet FindDeclarationRuleSet(Project project, RuleSetDeclaration decla
             return ruleSetSerializer.LoadRuleSet(ruleSetFilePath);
         }
 
-        private bool HasInclude(RuleSet source, RuleSet target)
+        private bool HasInclude(RuleSet source, string targetRuleSetFilePath)
         {
             Debug.Assert(Path.IsPathRooted(source.FilePath));
-            Debug.Assert(Path.IsPathRooted(target.FilePath));
+            Debug.Assert(Path.IsPathRooted(targetRuleSetFilePath));
 
             // The path in the RuleSetInclude could be relative or absolute.
             // If relative, we assume it's relative to the source ruleset file.
             var sourceDirectory = Path.GetDirectoryName(source.FilePath);
-            var canonicalTargetFilePath = Path.GetFullPath(target.FilePath);
```

Comment:
... and this is the only place the RuleSet.FilePath was accessed.

Reply:
Wow.  🤦